src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,25 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int minPassLength = 6;
+    private static final int adultAge = 18;
```

Comment:
It`s not a valid names for constant variables https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names